libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -617,9 +635,8 @@ class MapboxNavigation(
      *
      * @see unregisterEHorizonObserver
      */
-    @Deprecated("Temporarily no-op. Functionality will be reintroduced in future releases.")
     fun registerEHorizonObserver(eHorizonObserver: EHorizonObserver) {
```

Comment:
Could you add here the same narrative documentation as in `EH_v3 integration document`? Don't need to paste in all the code, but mention the flow and how the API should be used.

Reply:
done